packages/core/src/i18n/index.ts

```diff
@@ -0,0 +1,64 @@
+import i18next from 'i18next';
+import LanguageDetector from 'i18next-browser-languagedetector';
+import merge from 'lodash.merge';
+import { initReactI18next } from 'react-i18next';
+
+import type { i18nOptions } from '../types.js';
+import en from './en.json';
+import es from './es.json';
+
+export const languageLocalStorageKey = 'i18nextLng_dapp';
+
+export default function initialize(options?: i18nOptions): void {
+  const translationsJson: any = {
+    en: {
+      translation: merge(en, options?.en || {}),
+    },
+    es: {
+      translation: merge(es, options?.es || {}),
+    },
+  };
+
+  if (options) {
+    const customLocales = Object.keys(options).filter(
+      key => key !== 'en' && key !== 'es',
+    );
```

Comment:
```suggestion

export const languageLocalStorageKey = 'i18nextLng_dapp';

export default function initialize(options?: i18nOptions): void {
  const translationsJson: any = {
    en: {
      translation: merge(en, options?.en || {}),
    },
  };

  if (options) {
    const customLocales = Object.keys(options).filter(
      key => key !== 'en',
    );
```

Remove spanish from defaults, does not make sense on having it here if even our main dapp does not have translations yet.
Lets leave option to add it as custom one instead.